sergey.duev/task_4/src/main/java/com/hillel/BankConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package com.hillel;
+
+import com.hillel.models.Bank;
+import com.hillel.models.BankAccount;
+import com.hillel.models.Statement;
+import com.hillel.services.ProdBankAccountService;
+import com.hillel.services.TestBankAccountService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.EnableMBeanExport;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+@Configuration
+@ComponentScan("com.hillel.models")
+public class BankConfiguration {
+
+    @Bean
+    public ProdBankAccountService getProdBankAccountService(Bank bank) {
```

Comment:
You're duplicating beans. You already have @Service annotation for ProdBankAccountService

Use either @Bean or @Service, but not both annotations for the same component 
Notice that if you're using @Service, you should include service package in @ComponentScan